src/commands/runDraftTool/helper/runDraftHelper.ts

```diff
@@ -154,7 +151,7 @@ function getBinaryFileName() {
     if (operatingSystem === 'win32') {
         operatingSystem = 'windows';
         // Draft release v0.0.22 the file name has exe associated with it.
-        draftBinaryFile = `draft-${operatingSystem}-${architecture}.exe`;
+        draftBinaryFile = `draft-${operatingSystem}-${architecture}`;
```

Comment:
💡 This might need revert - we can catch this in big PR but I wonder how big it will be. 

since the binary is made via seperate name nomenclature, this will work `draftBinaryFile = draft-${operatingSystem}-${architecture}.exe;`  always.